index.html

```diff
@@ -319,6 +321,18 @@ <h2>Glossary</h2>
 
 
 
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="j">J</div>
+
+<p><dfn class="lint-ignore export">Jamo</dfn>. The basic unit used to form Hangul syllables, representing vowels and consonants in Korean. Besides the code points for Jamos, Unicode has 11,172 Hangul <em>syllables</em> that represent jamo combinations encoded as single characters. In practice, they are the main characters in actual use. (See also: <a href="https://www.unicode.org/faq/korean.html">Unicode Korean FAQ</a>)</p>
```

Comment:
I found the second sentence a little hard to follow when trying to think how someone unfamiliar with this stuff would read it. There are also some idiomatic bits of English I'd change. Perhaps:

> In addition to <a>code points</a> for Jamo, Unicode encodes 11,172 Hangul <em>syllables</em>. These represent combinations of jamo as single, pre-composed characters. In practice, such syllables are the main characters in actual use.

Reply:
Fixed.